modules/pensions/lib/pensions/monitor.rb

```diff
@@ -105,6 +105,22 @@ def track_create_success(in_progress_form, claim, current_user)
       Rails.logger.info('21P-527EZ submission to Sidekiq success', context)
     end
 
+    ##
+    # log POST claim save validation error
+    # @see PensionClaimsController
+    #
+    # @param claim [Pension::SavedClaim]
+    # @param current_user [User]
+    # @param e [Error]
+    #
+    def track_process_attachment_error(in_progress_form, claim, current_user)
+      StatsD.increment("#{CLAIM_STATS_KEY}.process_attachment_error")
+      Rails.logger.error('21P-527EZ process attachment error',
+                         { confirmation_number: claim&.confirmation_number, user_uuid: current_user&.uuid,
+                           in_progress_form_id: in_progress_form&.id, errors: claim&.errors&.errors,
+                           statsd: "#{CLAIM_STATS_KEY}.process_attachment_error" })
+    end
```

Comment:
need to update the docs
pass the error to the logging function and send the error message in the payload

Reply:
@wayne-weibel Where do you keep your docs on this?